### PR TITLE
updates to README, specifically figs

### PR DESCRIPTION
**Pull request by earroyo28**

### CODE REVIEW

**Comment by earroyo28** [comment]
I specifically wanted to see if I had added the figures correctly

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 The project TBD [Tau Be Damned] aims to use the amino acid sequence of a protein to identify whether it is disordered.
 
+![The TBD logo, nice isn't it?](https://github.com/Intrinsically-Disordered/TBD/blob/main/TBDlogo.png?raw=True)
```
Comment (jiaweih) [inline README.md]: It's actually not quite correct, as you can see from here: https://github.com/Intrinsically-Disordered/TBD/tree/updates#project-tbd-tau-be-damned

Comment (jiaweih) [reply README.md]: No worries. I'll modify this and the following images

File: README.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
-# main-project
 # Project TBD [Tau Be Damned]
 
 -------------------------
 
 The project TBD [Tau Be Damned] aims to use the amino acid sequence of a protein to identify whether it is disordered.
 
+![The TBD logo, nice isn't it?](doc/TBDlogo.png)
```
Comment (jiaweih) [inline README.md:7]: I have modified this to make it work. So I'll merge.

**Review by jiaweih — APPROVED** [review]

**Pull request merged**